components/network/ntp/patches/10-etcfix.patch

```diff
@@ -17,9 +17,13 @@ can go away.
  #  define	ALT_CONFIG_FILE "%windir%\\ntp.conf"
 --- libntp/audio.c.orig
 +++ libntp/audio.c
-@@ -66,7 +66,7 @@
- static struct audio_device device; /* audio device ident */
- #endif /* HAVE_SYS_AUDIOIO_H */
+@@ -62,11 +62,8 @@
+ /*
+  * Global variables
+  */
+-#ifdef HAVE_SYS_AUDIOIO_H
+-static struct audio_device device; /* audio device ident */
+-#endif /* HAVE_SYS_AUDIOIO_H */
```

Comment:
This change is not related to paths customization (see top of this file).  If this is really needed please add such a change as a new patch with a comment explaining why it is needed.  Thank you.

Reply:
removed